platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/geometry/LatLngQuad.java

```diff
@@ -0,0 +1,95 @@
+package com.mapbox.mapboxsdk.geometry;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.annotation.NonNull;
+
+import com.mapbox.mapboxsdk.exceptions.InvalidLatLngBoundsException;
+import com.mapbox.services.android.telemetry.constants.GeoConstants;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A geographical area representing a non-aligned rectangle
+ * <p>
+ * This class does not wrap valies to the world bounds
+ * </p>
+ */
+public class LatLngQuad implements Parcelable {
```

Comment:
In iOS, this is `MGLCoordinateQuad`, so I wanted to try and use similar terminology. Is there